src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -98,10 +98,10 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
         /// <summary>
         /// Gets or sets the Storage Size in GB for instance
         /// </summary>
-        [Parameter(Mandatory = true,
+        [Parameter(Mandatory = false,
             HelpMessage = "Determines how much Storage size to associate with instance.")]
         [ValidateNotNullOrEmpty]
-        public int StorageSizeInGB { get; set; }
+        public int? StorageSizeInGB { get; set; }
```

Comment:
You can make a required parameter optional. However, you can't make parameters nullable unless used in a patch-style operation. Meaning, null is used to clear an already set value.